readthedocs/proxito/tests/test_redirects.py

```diff
@@ -130,3 +130,26 @@ def test_proper_page_on_subdomain(self):
             'https://project.dev.readthedocs.io/en/latest/test.html',
         )
 
+    def test_slash_redirect(self):
+        host = 'project.dev.readthedocs.io'
+
+        url = '/en/latest////awesome.html'
```

Comment:
I suppose it will work the same way, but we could add a test with trailing slashes.

* `/en/latest/awesome.html///` should give 404
* `'/en/latest////` should work

Reply:
We aren't doing any 404 checking here, it will _always_ return a 302 for any path in `/`, since that's happening before any checking of the code. I added a test checking the `//` at the end tho.